packages/camera/camera_android_camerax/README.md

```diff
@@ -40,7 +46,7 @@ use cases, the plugin behaves according to the following:
 ### 240p resolution configuration for video recording
 
 240p resolution configuration for video recording is unsupported by CameraX,
-and thus, the plugin will fall back to 480p if configured with a
+and thus, the plugin will fall back to target 480p if configured with a
 `ResolutionPreset`.
```

Comment:
Shouldn't this be "with `ResolutionPreset.low`", rather than any `ResolutionPreset` as it says now?

Reply:
Oops good catch!